game/dota_addons/legion_td/scripts/npc/abilities/undeadbuilder/skills/viper_poison_attack_lua.txt

```diff
@@ -0,0 +1,70 @@
+    //=================================================================================================================
+    "viper_poison_attack_lua"
+    {
+        // Ability Technical Aspect
+        // base script folder	: scripts/vscripts
+        // base texture folder	: resource/flash3/images/spellicons
+        //-------------------------------------------------------------------------------------------------------------
+        "BaseClass"						"ability_lua"
+        "ScriptFile"					"abilities/undeadbuilder/viper_poison_attack_lua"
+        "AbilityTextureName"			"viper_poison_attack"
+        "FightRecapLevel"				"1"
+        "MaxLevel"						"1"
+        "precache"
+        {
+            "soundfile"	"soundevents/game_sounds_heroes/game_sounds_viper.vsndevts"
+            "particle"	"particles/units/heroes/hero_viper/viper_poison_attack.vpcf"
+            "particle"	"particles/units/heroes/hero_viper/viper_poison_debuff.vpcf"
+        }
+        // Ability General
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilityType"					"DOTA_ABILITY_TYPE_BASIC"
+        "AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_UNIT_TARGET | DOTA_ABILITY_BEHAVIOR_AUTOCAST | DOTA_ABILITY_BEHAVIOR_ATTACK"
+        "AbilityUnitTargetTeam"			"DOTA_UNIT_TARGET_TEAM_ENEMY"
+        "AbilityUnitTargetType"			"DOTA_UNIT_TARGET_HERO | DOTA_UNIT_TARGET_BASIC"
+        "AbilityUnitDamageType"			"DAMAGE_TYPE_MAGICAL"
+        "SpellDispellableType"			"SPELL_DISPELLABLE_YES"
+        "SpellImmunityType"				"SPELL_IMMUNITY_ENEMIES_NO"
+
+        // Ability Casting
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilityCastRange"				"600"
+        "AbilityCastPoint"				"0"
+
+        // Ability Resource
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilityCooldown"				"2"
+        "AbilityManaCost"				"0"
+
+        // Damage
+        //-------------------------------------------------------------------------------------------------------------
+
+        // Special
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilitySpecial"
+        {
+            "01"
+            {
+                "var_type"						"FIELD_FLOAT"
+                "duration"						"2"
+            }
+            "02"
+            {
+                "var_type"						"FIELD_FLOAT"
+                "damage"						"0.15"
+            }
+            "03"
+            {
+                "var_type"						"FIELD_INTEGER"
+                "bonus_movement_speed"			"-15"
+            }
+            "04"
+            {
+                "var_type"						"FIELD_INTEGER"
+                "bonus_attack_speed"			"-15"
+            }
+        }
+    }
+    
```

Comment:
Ciould you unindent this file? Such that it does not start with 4 white spaces? Just mark everything and press shift+tab